examples/Example_embed_shift_template_experiment.ipynb

```diff
@@ -0,0 +1,865 @@
+{
```

Comment:
<p>what does this mean? I think a little more explanation in the notebook header, even if it links back to the readme for that explanation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/ponyo/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>